evaluationBeta.html

```diff
@@ -14,4 +14,8 @@
     
     
 </body>
-</html>
\ No newline at end of file
+</html>
+
+
+Title : ["OOP, Design Patterns", "HTTP","HTML","CSS","Javascript","NodeJS"],
```

Comment:
Forgot to put the .ignore for anything with Beta in name xD this is a copy paste to see my old arrays in NewEvaluations. If u mean for the values I should to put at select name="${option[i].selectName}" I dunno still what will be used and how :D just tried to put something since I must not let it empty